### ISSUE TITLE
Revisit feature toggle/flag approach with nginx

### ISSUE THREAD

**Issue by patcon** [GCTC-NTGC/gc-digital-talent#4889]
<sup>ℹ️ [Figma (root)][figma] | [Sitemap][sitemap]</sup>
<sup>_This initial comment is collaborative and open to modification by all._</sup>

Spin-off of https://github.com/GCTC-NTGC/gc-digital-talent/issues/4583

## Description

The way we're handling feature flags/toggles is mainly based off features available in apache. We perhaps wish to rethink how this is happening in an nginx world, but it felt out of scope to @petertgiles in the main apache=>nginx conversion issues.

## Screenshot: Prototype
[![a screenshot of ...][image]][image-link]
<sup>[Click screenshot to visit Figma.][image-link]</sup>

## To Do
- [ ] research options and make a decision
- [ ] ...

   [image]: https://user-images.githubusercontent.com/305339/166570935-6425a615-82c7-4b2f-9840-5c7d698fa2ab.png
   [image-link]: https://example.com
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap

**Comment by petertgiles** [comment]
Can be removed if https://github.com/GCTC-NTGC/gc-digital-talent/issues/6387 goes well.

**Comment by mnigh** [comment]
@petertgiles can this issue be closed as not planned based on [your comment](https://github.com/GCTC-NTGC/gc-digital-talent/issues/6387#issuecomment-1803881685)?

**Comment by petertgiles** [comment]
No, I'd like to keep this.  As mentioned in the other comment, I'm not satisfied with our current approach.  There's some good info in here I'd like to keep, too.  I'm OK to ice it though.

**Comment by petertgiles** [comment]
Reviewed in refinement:
- a feature flag product is available in Azure
- current solution is working OK so a refactor is low value

**Comment by esizer** [comment]
This may be part of a future spike but we could leverage azue app config:

https://learn.microsoft.com/en-us/azure/azure-app-configuration/manage-feature-flags?tabs=azure-portal
